uber/models.py

```diff
@@ -1048,7 +1049,10 @@ def _status_adjustments(self):
     @presave_adjustment
```

Comment:
We'll need to add a new staffing adjustment, which is that if someone is unassigned from a department, they're also unmarked as trusted in that department.